Python/dynload_aix.c

```diff
@@ -145,8 +145,8 @@ aix_loaderror(const char *pathname)
         ERRBUF_APPEND("\n");
     }
     errbuf[strlen(errbuf)-1] = '\0';            /* trim off last newline */
-    pathname_ob = PyUnicode_FromString(pathname);
-    errbuf_ob = PyUnicode_FromString(errbuf);
+    pathname_ob = PyUnicode_DecodeFSDefault(pathname);
+    errbuf_ob = PyUnicode_DecodeFSDefault(errbuf);
     PyErr_SetImportError(errbuf_ob, NULL, pathname);
```

Comment:
It seems that ``pathname_ob`` should be passed here, not ``pathname``.

Reply:
Good catch, fixed.